server/utils/authentication-utils.ts

```diff
@@ -0,0 +1,35 @@
+export class AuthenticationUtils {
+  //Returns payload of JWT
+  static parseJwt(token: string) {
+    return JSON.parse(Buffer.from(token.split('.')[1], 'base64').toString());
+  }
+
+  //If token doesn't exist, or is expired, return as invalid
+  static jwtExpired(token: string | undefined) {
```

Comment:
Could rename this to isJwtExpired() or hasJwtExpired()
Or
isJwtValid() and flip the logic